portfolio/src/main/webapp/style.css

```diff
@@ -72,6 +72,7 @@ a {
 
 #projects {
   align-items: center;
+  background-color:var(--secondary);
```

Comment:
add a space after `:`, here and elsewhere

Reply:
Updated with commit ea2db77